tests/cpp-tests/Classes/LabelTest/LabelTestNew.cpp

```diff
@@ -933,6 +935,57 @@ std::string LabelFNTBounds::subtitle() const
     return "Testing bounding-box";
 }
 
+LabelFNTMultiFontAtlasNoRotation::LabelFNTMultiFontAtlasNoRotation()
+{
+    auto s = Director::getInstance()->getWinSize();
+
+    auto label1 = Label::createWithBMFont("fonts/helvetica-32.fnt", "This is Helvetica");
+    addChild(label1);
+    label1->setPosition(Vec2(s.width / 2, s.height / 3 * 2));
+
+    auto label2 = Label::createWithBMFont("fonts/geneva-32.fnt", "And this is Geneva", TextHAlignment::LEFT, 0, Rect(0, 128, 0, 0), false);
+    addChild(label2);
+    label2->setPosition(Vec2(s.width / 2, s.height / 3 * 1));
+}
+
+std::string LabelFNTMultiFontAtlasNoRotation::title() const
+{
+    return "New Label + Multi-BM Font Atlas Test1";
+}
+
+std::string LabelFNTMultiFontAtlasNoRotation::subtitle() const
+{
+    return "Using 2 .fnt definitions that share the same texture atlas.";
+}
+
+LabelFNTMultiFontAtlasWithRotation::LabelFNTMultiFontAtlasWithRotation()
+{
+    auto s = Director::getInstance()->getWinSize();
+
+    auto spriteCache = SpriteFrameCache::getInstance();
+    spriteCache->addSpriteFramesWithFile("fonts/bmfont-rotated-test.plist");
```

Comment:
Font resources for tests are missing

Reply:
I just realised that too, and updated them as you typed the comment. Sorry about that!

Reply:
It’s ok, I didn't realise the "Refresh" notice.